### ISSUE TITLE
IE11 Text rendering issues (unexpected capitalized letters)

### ISSUE THREAD

**Issue by webdevkit** [mathjax/MathJax#2484]
### Issue Summary

Some characters are rendering with the wrong capitalization in IE11 for certain scenarios. For example, the expected text label "Test" is rendering as "TeSt".

### Steps to Reproduce:

1. Go to the MathJax demo in IE11 (https://www.mathjax.org/#demo)
2. Enter in the following expression: `$$\text{SS1} + \text{Test}$$`

It appears that the second S in SS1 is causing the text glitch. If I change it to S1, then "Test" renders correctly. As soon as I add a second capital S in SS1 (or make it something like SuperSonic 1 with a second capital S), then the text gets funky.

In Chrome and Edge, the test text always renders as "Test" regardless of what is happening, so I think it is some issue with how IE11 is handling the fonts.

### Technical details:

* MathJax Version: 3.0.5
* Client OS: Windows 10
* Browser: Internet Explorer 11

### Supporting information:

<img width="1920" alt="Screen Shot 2020-07-29 at 2 46 15 PM" src="https://user-images.githubusercontent.com/8787457/88840345-4fb30800-d1aa-11ea-87cc-184d2cf7c230.png">

No console errors.

Default configuration.

**Comment by dpvc** [comment]
This may be related to #2440, which was for Microsoft Edge.  Apparently it is a bug in a Microsoft library.

For a work-around fo now, you could switch to SVG output rather than CommonHTML.

**Comment by webdevkit** [comment]
Thanks for the quick response, @dpvc. I was going down some rabbit hole about IE having issues with font-family names needing to match font filenames and IIS configuration hell trying to figure out how to fix it, but nothing was working

Appreciate the SVG suggestion. I did a quick test and confirmed that when I switch the output to SVG mode, IE does start rendering the characters as expected. I'm going to present my findings to the team tomorrow and likely move on with this as our solution. I don't see any specific need to use CHTML over SVG, so I'm guessing it doesn't matter much, whereas supporting our IE11 users does.

If you folks want to consider this a closed case since you are working on #2440, then feel free.
